### DISCUSSION TITLE
The 'GitHub Student Developer Pack' isn't accepting my ISIC card

### DISCUSSION THREAD

**Discussion by creeper6530** [community/community#67117] [GitHub Education]
### Hi everyone,

I tried to apply for the GitHub Student Developer Pack using my (valid) ISIC card as a proof, but got rejected.

The error message is following:
```
You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name exactly as it appears in your academic affiliation document plus a short bio. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.

Please make sure the document you uploaded includes your full name as it appears on your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile).
```

I'm from Czechia and my full name includes non-ASCII characters (`Ř` and `Ě`, to be precise), therefore I think the OCR isn't recognising them. The school's name and logo is in Czech as well, but the whole ID is in English.

How do I explain to the system that my name is illegible for English-tailored OCR?

**Comment by dharmendra-byte** [comment]
No bro #1

**Comment by gfb-47** [comment]
Did you take a photo of the ISIC App Screen?

**Reply by creeper6530** [reply]
No, the plastic card itself

**Comment by jehlijos** [comment]
I have the same problem, any updates in this issue?

**Reply by creeper6530** [reply]
None, I have given up on this and am waiting for a miracle

**Reply by jehlijos** [reply]
I did it on 5th attempt. You have to edit your name in Billing Information settings to match name on your ISIC. For instance i had my name in Billing Information settings with "c" instead of "č" on my ISIC card. I retyped it in github profile settings and made another attempt with ISIC photo this time successfully.

**Comment by joshbigmouse** [comment]
One thing you could try is making sure your GitHub profile name exactly matches the name on your ISIC card, even if you need to omit or simplify the special characters temporarily. You might also consider reaching out to GitHub support directly, explaining that your name contains Czech characters that the OCR may not be able to process. Including a screenshot of the error message could help clarify the situation.

**Reply by jehlijos** [reply]
GitHub support for this purpose is no longer supported. If you try to contact support with education authentication, you are informed that direct support by staff is no longer available.
